### PR TITLE
Netatmo: run parent refresh for modules in a scheduled task

### LINKED ISSUE #3132: Netatmo: Initializing handler takes more than 5000ms
```
2018-01-17 19:23:09.956 [WARN ] [ome.core.thing.internal.ThingManager] - Initializing handler for thing 'netatmo:NACamera:maison:camera' takes more than 5000ms.
2018-01-17 19:23:09.961 [WARN ] [ome.core.thing.internal.ThingManager] - Initializing handler for thing 'netatmo:NAWelcomePerson:maison:Laurent' takes more than 5000ms.
```
Only things relative to camera are apparently impacted.

### PR DESCRIPTION
**Pull request by lolodomo**
Fixes #3132

Signed-off-by: Laurent Garnier <lg.hc@free.fr>

### CODE REVIEW

**Comment by lolodomo** [comment]
@clinique for review

File: addons/binding/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/handler/NetatmoModuleHandler.java [hunk 1]
```diff
@@ -43,7 +46,17 @@ protected NetatmoModuleHandler(@NonNull Thing thing) {
     @Override
     public void initialize() {
         super.initialize();
-        requestParentRefresh();
+        refreshJob = scheduler.schedule(() -> {
+            requestParentRefresh();
+        }, 5, TimeUnit.SECONDS);
```
Comment (martinvw) [inline addons/binding/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/handler/NetatmoModuleHandler.java]: Every 5 seconds feels often, why?

Comment (lolodomo) [reply addons/binding/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/handler/NetatmoModuleHandler.java]: No it is only one one shot task delayed of 5 seconds.

Comment (martinvw) [reply addons/binding/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/handler/NetatmoModuleHandler.java]: Of course, thanks!

**Comment by martinvw** [comment]
@clinique should I wait for you?

**Comment by clinique** [comment]
No, you can proceed. If a fixed delay of 5sec does not show good acceptance from users, we'll make it evolve as a config parameter.

**Pull request merged**